services/catalog/openapi.json

```diff
@@ -979,6 +1089,11 @@
               }
             ],
             "description": "The node's state object"
+          },
+          "bootOptions": {
+            "title": "Bootoptions",
+            "type": "object",
+            "description": "Some services provide alternative parameters to be injected at boot time. The user selection should be stored here, and it will overwrite the services's defaults."
```

Comment:
how come I see bootOptions here? Uhm... I wonder what happened.

Reply:
well this is generated with the ```make openapi-spec``` recipe.. not sure when it was run last time...